Tests/TestCesiumOriginShift.cs

```diff
@@ -97,4 +97,96 @@ public IEnumerator UsesGeoreferenceTransformForActivationCheck()
 
         Assert.AreEqual(subScene.isActiveAndEnabled, true);
     }
+
+    [UnityTest]
+    public IEnumerator UsesActivationDistanceProperty()
+    {
+        double3 baseEcef = new double3(-2694019.41, -4297353.83809221, 3854717.9087824);
+
+        GameObject goGeoreference = new GameObject("Georeference");
+        CesiumGeoreference georeference = goGeoreference.AddComponent<CesiumGeoreference>();
+        georeference.ecefX = baseEcef.x;
+        georeference.ecefY = baseEcef.y;
+        georeference.ecefZ = baseEcef.z;
+
+        GameObject goOriginShift = new GameObject("OriginShifter");
+        goOriginShift.transform.parent = goGeoreference.transform;
+        CesiumGlobeAnchor globeAnchor = goOriginShift.AddComponent<CesiumGlobeAnchor>();
+
+        CesiumOriginShift originShift = goOriginShift.AddComponent<CesiumOriginShift>();
+        originShift.useActivationDistance = true;
+        originShift.activationDistance = 5000;
+
+        yield return null;
+
+        globeAnchor.positionGlobeFixed = baseEcef + new double3(4999.0, 0, 0);
+
+        yield return null;
+
+        Assert.AreEqual(baseEcef.x + 4999.0, globeAnchor.positionGlobeFixed.x);
+        Assert.AreEqual(baseEcef.x + 0, georeference.ecefX);
```

Comment:
Nitpick but we can just simplify this
```suggestion
        Assert.AreEqual(baseEcef.x, georeference.ecefX);
```